metadata/boundaries/municipal_boundaries

```diff
@@ -0,0 +1,80 @@
+# Title
+
+Utah Municipal Boundaries
+
+## ID
+
+6b60c759-74d6-4533-93f5-fed8cf32a748
+
+## Brief Summary
+
+Statewide polygon dataset of municipal boundaries for cartography and approximate boundary identification.
+
+## Summary
+
+Utah municipal boundaries defined by County Recorders' offices and certified by the Lieutenant Governor's Office. Updated boundaries from county authorities are aggregated by UGRC to produce this dataset.
+
+## Description
+
+### What is the dataset?
+
+This dataset represents the 250+ municipalities in Utah as polygons that include information such as:
+
+- Whether or not the municipality is a county seat
+- Municipality population as of the 2020 Census
+- Estimated population of the municipality as of January 1, 2022
```

Comment:
**thought**: If this is based on census i would maybe combine this with the bullet above?
```suggestion
- Estimated population of the municipality as of January 1, 2022
```

Reply:
I'm actually not too sure how the 2022 population estimate was calculated, but I would presume the census was involved in some way. In the dataset, there are separate fields for the census population and the population estimate, POPLASTCEN and POPLASTEST. I think it's worth including in the metadata that the estimate (POPLASTEST) is from 2022, rather than 2020, so that a person knows what the difference between the two fields is.

Reply:
I found this in an old mike heagin blog post... doesn't clear anything up really though.

> This quarter we got the latest (July 1, 2022) population estimates for the towns and cities and the `POPLASTESTIMATE` field reflects those changes.

Reply:
Huh. I wonder if the POPLASTEST field is updated periodically, then? @DenisePeterson do you know how often these population estimates are released, and how often we apply them to this dataset?

Reply:
@gregbunce do you have more info on this?

Reply:
It looks like POPLASTEST matches the 2022 incorporated places numbers from https://www.census.gov/data/tables/time-series/demo/popest/2020s-total-cities-and-towns.html#v2022. There's a methodology doc linked there for the Census Population Estimates Program (PEP)

Reply:
Fantastic, thank you for finding that! I'll clarify that in the metadata.